Docs for creating custom build helpers (#1799)
conan-io/docs: extending/custom_build_helper.rst

.. _custom_build_helper:

Creating a custom build helper for Conan
----------------------------------------

> If Conan has not a build helper for the build tool you are using, you can create a custom build helper

Reviewer: It that "has not" correct?

```suggestion
If Conan doesn't have a build helper for the build tool you are using, you can create a custom build helper
```

> with the help of :ref:`python_requires`. You can create a package defining the build helper for that

Reviewer: A lot of "help" here.

```suggestion
with the :ref:`python_requires`. You can create a package defining the build helper for that
```
